pubspec.yaml

```diff
@@ -11,10 +11,12 @@ environment:
 
 dependencies:
   collection: ^1.15.0
-  meta: ^1.3.0
   logging: ^1.0.1
+  meta: ^1.3.0
   test: ^1.17.3
```

Comment:
The `SimCompare` utility uses `test`'s `expect` function, and some other users of ROHD like to reuse `SimCompare`, which means it needs to be available as a "real" dependency instead of just dev.  It should probably be exposed through `rohd.dart`.  I updated #134 to mention SimCompare too.